app/models/booking.rb

```diff
@@ -1,7 +1,18 @@
 class Booking < ApplicationRecord
   belongs_to :user
   belongs_to :adventure
-  validates :start_date, presence: true
-  validates :end_date, presence: true
-end
+  validates :start_date, :end_date, presence: true
+  validates :nr_of_booked_guests, presence: true
+  validate :end_date_after_start_date
+
+  private
+
+  def end_date_after_start_date
+    return if end_date.blank? || start_date.blank?
```

Comment:
Hey @jeegun - what are you doing with the return if here?  are we returning anything?

Reply:
@raph-dav good catch. was just looking into that to create another error. will fix.